src/main/java/com/crud/cadastrousuario/domain/model/Book.java

```diff
@@ -49,6 +58,14 @@ public class Book {
     private List<Author> authors = new ArrayList<>();
 
 
+
+    @OneToMany(mappedBy = "bookID")
```

Comment:
Esteja ciente da estratégia de carregamento padrão do Hibernate. Por padrão, @OneToMany usa a estratégia de carregamento "lazy", o que significa que os objetos relacionados não são carregados do banco de dados até que sejam explicitamente acessados. Isso pode levar a problemas de desempenho se não for gerenciado corretamente.

Reply:
Optei por usar modo eager para carregar todos os dados que vão ser usados, você recomenda o uso do modo lazy nesse caso  mano?